merge_sort/Java/merge_sort.java

```diff
@@ -0,0 +1,56 @@
+import java.util.Scanner;
+import java.util.Arrays;
+
+class merge_sort {
+
+    public static void main(String args[]) {
+
+        int intArray[] = {9, 5, 4, 6, 1, 2, 3, 1, 2, 3};
+        System.out.println("The unsorted array is:");
+        System.out.println(Arrays.toString(intArray));
+
+        mergesort(intArray);
+
+        System.out.println("The sorted array is:");
+        System.out.println(Arrays.toString(intArray));
+    }
+
+    static void mergesort(int intArray[]) {
```

Comment:
Please add comments for this method. You can understand how much code comments are necessary for the maintenance.

Reply:
Done.